pkg/services/mssql/database_fe_provision.go

```diff
@@ -0,0 +1,88 @@
+package mssql
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/open-service-broker-azure/pkg/service"
+	uuid "github.com/satori/go.uuid"
+)
+
+func (d *databaseFeManager) GetProvisioner(
+	service.Plan,
+) (service.Provisioner, error) {
+	return service.NewProvisioner(
+		service.NewProvisioningStep("preProvision", d.preProvision),
+		service.NewProvisioningStep("getDatabase", d.getDatabase),
+		service.NewProvisioningStep("deployARMTemplate", d.deployARMTemplate),
```

Comment:
What resource will `deploymentARMTemplate` creates? I think the service `database-from-existing` shouldn't create any resource?

Reply:
It creates nothing but only the deployment. It is for update api to use the deployment so that the implementation uses the same pattern like other services (call armDeployer.Update()).